internal/handlers/utils.go

```diff
@@ -0,0 +1,129 @@
+package handlers
+
+import (
+	"context"
+	"fmt"
+	"github.com/ydb-platform/ydb-go-genproto/protos/Ydb"
+	"github.com/ydb-platform/ydb-go-genproto/protos/Ydb_Operations"
+	"github.com/ydb-platform/ydb-go-sdk/v3"
+	"go.uber.org/zap"
+	"time"
+	"ydbcp/internal/config"
+	"ydbcp/internal/connectors/client"
+	"ydbcp/internal/connectors/db"
+	"ydbcp/internal/types"
+	"ydbcp/internal/util/xlog"
+)
+
+func DeadlineExceeded(createdAt time.Time, config config.Config) bool {
```

Comment:
should it be a public function or is the package level enough?

Reply:
made package level